active/tab/tabcontrol.md

```diff
@@ -0,0 +1,401 @@
+* Modern browsers support selecting tabs when pressing CTRL and a number. Consider handling Ctrl+1 through 8, which should select that number Tab. Additionally, you can consider handling Ctrl+9 which selects the last tab (regardless of the number of tabs).
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the Tab. 
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+### TabView properties, events, methods
+
+| Property | Type | Description |
+|:-------- |:---- |:----------- |
+| AddButtonCommand | ICommand | Gets or sets the command to invoke when the Add button is tapped. |
+| AddButtonCommandParameter | object | Gets or sets the parameter to pass to the command for the Add button. |
+| AddTabButtonVisibility | Visibility | Determines if the plus button appears to the right of the tab strip |
+| CanDragTabs | bool | Gets or sets a value that indicates whether tabs in the collection can be dragged. Default is true. |
+| CanReorderTabs | bool | Gets or sets a value that indicates whether tabs in the collection can be reordered through user interaction. Default is true. |
+| ItemsSource | object | Gets or sets an object source used to generate the tabs. |
+| Items | IVector<object> | Gets the collection used to generate the tabs. |
+| ItemTemplate | DataTemplate | Gets or sets the DataTemplate used to display each item. |
+| ItemTemplateSelector | DataTemplateSelector | Gets or sets a reference to a custom DataTemplateSelector logic class. The DataTemplateSelector referenced by this property returns a template to apply to items. |
+| SelectedIndex | Int32 | Gets or sets the index of the selected item. |
+| SelectedItem | object | Gets or sets the selected item. |
+| TabStripHeader | object | Content to the left of the tab strip. |
+| TabStripFooter | object | Content to the right of the tab strip. |
+| TabWidthMode | enum | Specifies how the tabs should be sized. Values are {Actual, Equal}. Default is Actual. |
+
+| Event | Description |
+|---|---|
+| AddTabButtonClick | Fires when the + button is clicked. |
+| SelectionChanged | Occurs when the currently selected tab changes. |
+| TabClosing | Fires when a tab is about to be closed. Can be cancelled to prevent closure. |
+| TabDraggedOutside | Fires when a Tab is dragged outside of the Tab bar. |
+
+| Method | Description |
+|---|---|
+| DependencyObject ContainerFromItem(object item) | Returns the generated container for the specified item. |
+| DependencyObject ContainerFromIndex(Int32 index) | Returns the generated container for the specified index. |
+
+### TabViewItem properties and events
+
+| Property | Type | Description |
```

Comment:
Once we have the IDL, we should remove this Type column. It's not entirely correct, and once we have the IDL it just creates the overhead of staying in sync.